app/models/operatingsystems/vsphere.rb

```diff
@@ -0,0 +1,23 @@
+class Vsphere < Operatingsystem
+  PXEFILES = {:kernel => "mboot.c32", :initrd => ""}
+
+  class << self
+    delegate :model_name, :to => :superclass
+  end
+
+  def mediumpath(host)
+    medium_uri(host).to_s
+  end
+
+  def pxe_type
+    "kickstart"
+  end
+
```

Comment:
This needs an `available_loaders` method too to list all of the possible PXE loaders